### ISSUE TITLE
Update submission instructions on Meetup page

### ISSUE THREAD

**Issue by laras126** [GothamSass/chrysler#93]
In "About" roll over it currently says to submit to the Wufoo form, and this link was just in a tweet. Also, can a non-collaborators add labels to an issue? Maybe I'm missing that.

**Comment by adekunleoduye** [comment]
@laras126 Good point....we have to get that updated ASAP...thanks for pointing that out!
